tests/fixtures/sort/multiple_decimals_general.txt

```diff
@@ -32,4 +32,6 @@ CARAvan
 8.013
 
 000
-
+  NaN
```

Comment:
is that expected?

Reply:
I added a test case for NaN. The expected output is in tests/fixtures/sort/multiple_decimals_general.expected